src/v2/Apps/Consign/Routes/SubmissionFlow/Utils/__tests__/createConsignSubmission.jest.ts

```diff
@@ -27,13 +33,27 @@ describe("createOrUpdateConsignSubmission", () => {
       submission
     )
 
-    expect(createConsignSubmissionMutation).toHaveBeenCalled()
-    expect(createConsignSubmissionMutation).toHaveBeenCalledWith(
+    expect(mockCreateMutation).toHaveBeenCalled()
+    expect(mockCreateMutation).toHaveBeenCalledWith(
       relayEnvironment,
       submission
     )
-    expect(result).toEqual("123")
+    expect(result).toEqual("111")
   })
 
-  it.skip("updates submission", async () => {})
+  it("updates submission", async () => {
+    submission = { artistID: "artistId", id: "123" }
+
+    const result = await updateConsignSubmissionMutation(
+      relayEnvironment,
+      submission
+    )
+
+    expect(mockUpdateMutation).toHaveBeenCalled()
+    expect(mockUpdateMutation).toHaveBeenCalledWith(
+      relayEnvironment,
+      submission
+    )
+    expect(result).toEqual("222")
```

Comment:
**question:** if the submission ID is `"222"` instead of `"123"` then I think that a new submission was created, instead of updating an existing one - is that correct?

Reply:
It just for the test. I mocked the backend response. I suppose the backend has logic to determine which id he'll return.

Reply:
I see it now. Thanks for the clarification!